release-notes/opensearch-security.release-notes-2.0.0.0-rc1.md

```diff
@@ -0,0 +1,24 @@
+## 2022-04-20 Version 2.0.0.0
+
+Compatible with OpenSearch 2.0.0-rc1
+
+### Enhancements
+
+* Incremented version to 2.0-rc1. ([#1764](https://github.com/opensearch-project/security/pull/#1764))
```

Comment:
calling this an enhancement is a bit of a stretch :) this feels more of a maintenance issue.